scripts/case_selection/perform_case_selection.Rmd

```diff
@@ -7,6 +7,7 @@ params:
   phase: x
```

Comment:
Is this the only place in the case section process that we can see the versions of things that were ran?

Reply:
Yup, as my understanding.

Reply:
There's also provenance, which should link back to the different versions.